spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -391,7 +391,7 @@ def main(argv: Sequence[str]):
                        f"not be displayed. To use vertebral level information, you may need to run "
                        f"`sct_warp_template` to generate the appropriate level file in your working directory.")
         if normalize_pam50:
-            raise FileNotFoundError("The vertebral level file must exist to use -normalize-PAM50.")
+            parser.error("Option '-normalize-PAM50' requires option '-vertfile'.")
```

Comment:
Thanks for this fix! I very much agree with this change.

It looks like the failing test is due to the error type changing, so now the error no longer gets caught by the test. I can update the test accordingly so that the new error type is caught.

Reply:
Great, thank you for the test update!